docs/Developers/Newcomers_Guide.rst

```diff
@@ -87,7 +87,23 @@ Step 1. Meet the Community!
 
 To get started, the first step is to meet the community. We use gitter to
 communicate, and there the helpful community will guide you.
-Join us at `coala gitter <https://coala.io/chat>`_.
+Gitter is instant messaging service used by developers and users of GitHub.
+Gitter uses chatrooms, where developers can join in and can talk about particular topic.
+Coala has several chatrooms. Here are a few of them. 
+
+* `coala <https://gitter.im/coala/coala>`_
+* `gsoc <https://gitter.im/coala/coala/gsoc>`_
+* `cobot-test <https://gitter.im/coala/cobot-test>`_
+* `coala-bears <https://gitter.im/coala/coala-bears>`_
+* `workshop <https://gitter.im/coala/coala/workshops>`_
+* `freelancers <https://gitter.im/coala/freelancers>`_
```

Comment:
This listing could be improved with more info and a less random selection of rooms. There are two types of rooms: repo specific, like coala (although coala is also the main channel for the org) and coala-bears and then there are topic channels like conferences, gsoc and offtopic.